amethyst_ui/src/prefab.rs

```diff
@@ -343,6 +343,19 @@ pub enum UiImageLoadPrefab {
     PartialTexture(TexturePrefab, f32, f32, f32, f32),
     /// Solid color image
     SolidColor(f32, f32, f32, f32),
+    /// 9-Slice image
+    NineSlice {
+        x_start: i32,
```

Comment:
Should likely use `u32`s for all of these

Reply:
you are correct. Fixed this here and in the UiImage definition.